packages.yml

```diff
@@ -1,3 +1,4 @@
 packages:
 
-  - local: ../dbt_klaviyo_source
\ No newline at end of file
+  - package: fivetran/klaviyo_source
+    version: [">=0.1.0","<0.2.0"]
```

Comment:
Can we revert back to git to see a succesful circle run in the commit history before merging? Then we can switch back to the release before release.